### DISCUSSION TITLE
no binary found in 2.2.0

### DISCUSSION THREAD

**Discussion by hy2014** [apache/kvrocks#1152] [Q&A]
just as title,  it is hard for user build on centos7, many dep need to upgrade, it may not welcome for users.

**Comment by tisonkun** [comment]
@hy2014 do you want a prebuilt RPM installer for Kvrocks 2.2.0?

**Comment by PragmaTwice** [comment]
I think the provided docker image is more general and easy to use. It is hard to build binary packages for all Linux distributions.

And I want to talk more about binary packages like rpm or deb. Actually even we build some rpm or deb packages, it is still hard to be available to all distributions and distribution versions (which support this package format) to use it. For example, the shared-linked glibc version is well-known hard to handle.

So I prefer to encourage users to use the docker image or build kvrocks on the environment where you will really use it.
